portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,17 +35,44 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-  private ArrayList<String> messages = new ArrayList<String>(Arrays.asList(
-      "Welcome to my page!", "Porfolio time!", "This is my portfolio"
-  ));
+  private ArrayList<String> comments = new ArrayList<String>();
+  private int numComments = 3;
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
doGet and doPost methods are getting long, can you add a sentence or two to describe what they are trying to do?